targets/csharp/source/PlayFabSDK/source/Qos/PlayFabQosApi.cs

```diff
@@ -0,0 +1,139 @@
+﻿namespace PlayFab.QoS
```

Comment:
This file needs to be wrapped in:
#if !DISABLE_PLAYFABCLIENT_API && !DISABLE_PLAYFABENTITY_API
#endif

(And as I think about it, every new file you have added here.)